Solutions/Azure Active Directory/Analytic Rules/Brute Force Attack against GitHub Account.yaml

```diff
@@ -43,7 +43,7 @@ query: |
     | summarize FailedLoginsCountInRunTime = count() by User = Identity, UserPrincipalName, bin(TimeGenerated, BinTime), Type
   ) on UserPrincipalName 
   | where FailedLoginsCountInRunTime > LearningThreshold
```

Comment:
We could probably vastly simplify this with use of imAuthentication and series_decompose_anomalies

Reply:
I totally agree. @oshezaf Should we build the query using imAuthentication function for now and change it to _Im_Authentication once we complete the deployment of Authentication parsers? Or should we make this change post deployment?
@petebryan For series_decompose_anomalies, should we consider only github.com or all the logs? I will take the first stab at it once you confirm.

Reply:
I would say just GitHub, that way we keep the purpose of this query and also I think if we went larger we might have perf issues.

Reply:
@petebryan Updated the query with series_decompose_anomalies